src/core/utils/cli-config.ts

```diff
@@ -76,10 +92,10 @@ export async function getConfigFileOptions(configName: string | undefined, confi
     }
 
     const [configName, config] = Object.entries(cliConfig.configurations)[0];
-    printConfigMsg(configName, configFilePath);
+    printConfigMsg(configName, resolvedConfigFilePath);
     currentSwaCliConfigFromFile = {
       name: configName,
-      filePath: configFilePath,
+      filePath: resolvedConfigFilePath,
```

Comment:
Suggestion:

You can handle the config path fallback logic in the beginning of this function, so that you won't need to change `configFilePath` to `resolved*` all the way down here.